packages/cubejs-dremio-driver/driver/DremioQuery.js

```diff
@@ -92,6 +99,33 @@ class DremioQuery extends BaseQuery {
   wrapSegmentForDimensionSelect(sql) {
     return `IF(${sql}, 1, 0)`;
   }
+
+  /**
+   * The input interval with (possible) plural units, like "1 hour 2 minutes", "2 year", "3 months", "4 weeks", "5 days", "3 months 24 days 15 minutes", ...
+   * will be converted to Dremio dialect.
+   * @see https://docs.dremio.com/24.3.x/reference/sql/sql-functions/functions/DATE_ADD/
```

Comment:
Looking at the docs, I see that Dremio supports more intervals for timestamps.
```
DATE_ADD(timestamp_expression string, time_interval interval) → timestamp
timestamp_expression: A string-formatted timestamp in the format 'YYYY-MM-DD HH24:MI:SS'.
time_interval: A CAST of a number to one of these intervals: SECOND, MINUTE, HOUR, DAY, MONTH, YEAR.
```

So i think they should be supported here too :)